src/main/java/hw5/page/UserTablePage.java

```diff
@@ -0,0 +1,92 @@
+package hw5.page;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import java.util.List;
+
+public class UserTablePage {
+
+    @FindBy(css = "#user-table tr")
+    private List<WebElement> tableRows;
+    @FindBy(name = "log-sidebar")
+    public WebElement logSidebar;
+
+    private By selectBy = By.tagName("select");
+    private By usernameBy = By.tagName("a");
+    private By descriptionBy = By.cssSelector(".user-descr span");
+    private By checkboxBy = By.cssSelector("input");
+    private WebDriver driver;
+
+    public UserTablePage(WebDriver driver) {
+        PageFactory.initElements(driver,this);
+        this.driver = driver;
+    }
+
+    public String getTitle() {
+        return driver.getTitle();
+    }
```

Comment:
In genereal these constructor and method could be extracted to the `Base` class

Reply:
I will make BasePage class for this reason.